packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -42,66 +47,73 @@ export class Relayer {
     readonly account: string
   ) {}
 
-  // TODO: consider refactoring this further into multiple methods. One to build the object of relayableDeposits and one
-  // that processes relayableDeposits and calls respective relaying functions.
-  async checkForPendingDepositsAndRelay() {
+  async checkForPendingDepositsAndRelay(): Promise<undefined> {
     this.logger.debug({ at: "Relayer", message: "Checking for pending deposits and relaying" });
-    const relayableDeposits: { [key: string]: [{ status: ClientRelayState; deposit: Deposit }] } = {};
-    for (const l1Token of this.whitelistedRelayL1Tokens) {
-      this.logger.debug({ at: "Relayer", message: "Checking relays for token", l1Token });
-      // TODO: consider limiting how far back we look in this call size.
```

Comment:
@chrismaree I think this comment can be ignored now since we pass an `l2StartBlock` into the L2 relay client. We can also consider adding an `l1StartBlock` but I haven't had issues querying L1 events.